emulators/qemu/Portfile

```diff
@@ -53,21 +53,23 @@ patchfiles-append       patch-qemu-configure.diff
 patchfiles-append       patch-qemu-punchhole.diff
 
 # "cp: illegal option -- a"
-patchfiles-append       patch-qemu-scripts-entitlements.diff
+# disabled because it does not appear to be needed for qemu 7.0.0
```

Comment:
If not needed, remove the lines and the patchfiles.

Reply:
Hi @ryandesign,

I have made these changes to my commit.

Thanks,

-ranga

Reply:
What about the other two?

Reply:
Hi @ryandesign,

I have removed them.

Thanks,

-ranga